src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -1796,6 +1796,11 @@ export function getCnaQueries(
         && vertSelection.entrezGeneId !== undefined) {
         queries.push({entrezGeneId: vertSelection.entrezGeneId});
     }
+    if (vertSelection.dataType === AlterationTypeConstants.COPY_NUMBER_ALTERATION
+        && horzSelection.dataType === AlterationTypeConstants.GENERIC_ASSAY
```

Comment:
@pvannierop @ngocnn1104 similar to other PR, i just want to make sure that we are not unnecessarily equating GENERIC_ASSAY with treatment type.   will this require refactoring down the line?